tests/components/smarttub/test_controller.py

```diff
@@ -0,0 +1,37 @@
+"""Test the SmartTub controller."""
```

Comment:
Move these tests to `test_init.py` and don't access the controller directly. Assert the config entry state and api calls.